skimage/morphology/misc.py

```diff
@@ -124,3 +127,81 @@ def remove_small_objects(ar, min_size=64, connectivity=1, in_place=False):
     out[too_small_mask] = 0
 
     return out
+
+def remove_small_holes(ar, min_size=64, connectivity=1, in_place=False):
+    """Remove connected components smaller than the specified size within a 
+       larger connected object.
```

Comment:
The convention for these is single line, fitting within 80c including indent. Maybe, `remove contiguous holes smaller than the specified size`?